app/src/main/java/com/andrew/tasky/presentation/event_detail/EventDetailFragment.kt

```diff
@@ -106,455 +98,402 @@ class EventDetailFragment : Fragment(R.layout.fragment_event_detail) {
+            endTimeAndDateLayout.timeButton.isEnabled = !isAttendee
+            endTimeAndDateLayout.timeButton.isVisible = !isAttendee
+            endTimeAndDateLayout.dateTextView.isEnabled = !isAttendee
+            endTimeAndDateLayout.dateButton.isEnabled = !isAttendee
+            endTimeAndDateLayout.dateButton.isVisible = !isAttendee
+            endTimeAndDateLayout.timeTextView.setOnClickListener {
+                showTimePickerDialog(it)
+            }
+            endTimeAndDateLayout.timeButton.setOnClickListener {
+                showTimePickerDialog(it)
+            }
+            endTimeAndDateLayout.dateTextView.setOnClickListener {
+                showDatePickerDialog(it)
+            }
+            endTimeAndDateLayout.dateButton.setOnClickListener {
+                showDatePickerDialog(it)
+            }
 
-            collectLatestLifecycleFlow(viewModel.description) { description ->
-                descriptionTextView.text = description
+            reminderLayout.reminderTextView.setOnClickListener {
+                showReminderOptionsPopupMenu(it)
+            }
+            reminderLayout.reminderButton.setOnClickListener {
+                showReminderOptionsPopupMenu(it)
+            }
+
+            attendeesLayout.addAttendeeButton.isVisible = !isAttendee
+            attendeesLayout.addAttendeeButton.setOnClickListener {
+                showAddAttendeeDialog()
+            }
+            attendeesLayout.allButton.setOnClickListener {
+                viewModel.setAttendeeFilterType(EventDetailViewModel.AttendeeFilterTypes.ALL)
+            }
+            attendeesLayout.goingButton.setOnClickListener {
+                viewModel.setAttendeeFilterType(EventDetailViewModel.AttendeeFilterTypes.GOING)
+            }
+            attendeesLayout.notGoingButton.setOnClickListener {
+                viewModel.setAttendeeFilterType(EventDetailViewModel.AttendeeFilterTypes.NOT_GOING)
             }
 
             if (!isAttendee) {
-                collectLatestLifecycleFlow(viewModel.isInEditMode) { isEditing ->
-                    editDescriptionButton.isVisible = isEditing
-                    editDescriptionButton.isEnabled = isEditing
-                    descriptionTextView.isEnabled = isEditing
-                }
-                editDescriptionButton.setOnClickListener {
-                    navigateToEditFragment(EditType.DESCRIPTION)
-                }
-                descriptionTextView.setOnClickListener {
-                    navigateToEditFragment(EditType.DESCRIPTION)
+                btmActionTvBtn.deleteAgendaItemButton.text = String.format(
+                    resources
+                        .getString(R.string.delete_agenda_item_button),
+                    getString(R.string.event)
+                ).uppercase()
+            }
+            btmActionTvBtn.deleteAgendaItemButton.setOnClickListener {
+                if (isAttendee) {
+                    viewModel.switchAttendingStatus()
```

Comment:
In this, an attendee does see Leave event instead of Delete event. Also, I thought bases on the mockup and requirements pdf that an attendee could leave and easily join back with a Join event button. 
I can rename deleteAgendaItem button to bottomTvBtn instead

Reply:
deleteAgendaItem button sounds good to me, but an attendee can only join back if the owner adds them again